handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+|<img src="../../img/mdb-rs.png" width="1500px"/>| Multi node with MongoDB replica sets | Here SystemLink is hosted on a dedicated server and MongoDB is hosted on 3 or more servers. **This is the recommended configuration for all production deployments including small and moderately sized deployments.** This configuration enables greater scale (>50 managed nodes, >1000 test steps and results daily). This configuration provides for greater mitigation of data loss with redundant replica sets. Refer to [MongoDB documentation](https://docs.mongodb.com/manual/tutorial/deploy-replica-set/) for setting up and hosting a MongoDB instance with replica sets. |
+|<img src="../../img/mdb-atlas.png" width="1500px"/>| Multi node with MongoDB Atlas | Here SystemLink is hosted on a dedicated server and connects to a MongoDB Atlas cluster. This is recommended for large scale deployments (>200 managed node, 10,000 test steps and results daily). MongoDB Atlas also simplifies database provisioning, operation, backup and restore. These features may be desireable at any scale of deployment. Refer to [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) for details on this service. |
+
+### Changing the index cache size for *single node* deployments
+
+To constrain the amount of memory used by MongoDB, the default installation of SystemLink caps the available memory for the database's index at 1GB. This can lead to high CPU usage if the index grows beyond 1GB and must be paged to disk. If you are using the Test Insights or Asset Management modules, NI recommends increasing the available memory for the index to 8GB. The following steps can be used to increase the available memory for the MongoDB index.
+  
+1. With a user with administrator privileges, log into the desktop of the SystemLink app server.
+1. Open **NI SystemLink Server Configuration**.
+1. Navigate to **NoSqlDatabase**.
+1. Under **The NoSqlServer is launched and managed by SystemLink**, adjust the value in **The cache size of the database in Gigabytes (GB)**.
+1. Click **Apply**.
+
+!!!note
+    Refer to [Sizing a SystemLink server](https://www.ni.com/documentation/en/systemlink/latest/setup/sizing-a-systemlink-server/) for guidance on the CPU, memory, and disk needed for SystemLink single node deployments.
+
+### Connecting to MongoDB in a Multi Node Configuration
+
+SystemLink supports connecting to standalone MongoDB instances and replica sets. While SystemLink can connect and use a sharded MongoDB cluster (`mongos`) it will not take full advantage of the capabilities enabled by sharded clusters.
```

Comment:
Since this is introducing the next 3 subsections, something more direct/overview-y could help orient people here. 'SystemLink supports three types of multi-node configuration. You can configure SystemLink to connect to a standalone instance, replica sets, or MongoDB Atlas.'

Reply:
Good call. I've included this overview statement.